controllers/openstack_ansibleee_controller.go

```diff
@@ -55,10 +56,14 @@ const (
 type OpenStackAnsibleEEReconciler struct {
 	client.Client
 	Kclient kubernetes.Interface
-	Log     logr.Logger
 	Scheme  *runtime.Scheme
 }
 
+// GetLog returns a logger object with a prefix of "conroller.name" and aditional controller context fields
+func GetLog(ctx context.Context) logr.Logger {
+	return log.FromContext(ctx).WithName("Controllers").WithName("OpenStackAnsibleEE")
```

Comment:
yes, but the ask from some devs was to have the naming upfront also at the start of the line , so that the parsing is more human readable: 

how it would lokk after:
2023-09-12T23:11:48Z    INFO    Controllers.OpenStackAnsibleEE    starting manager {....reconcile details..."reconcileID": "512a4d9b-f31d-4fa4-a4cc-cd6c13e4455d"}